### PR TITLE
add check: route is consistent with selected PO

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by jgrandguillaume** [comment]
:+1:  That's perfect

File: sale_quotation_sourcing/model/sale_order.py [hunk 1]
```diff
@@ -71,6 +76,14 @@ class SaleOrderLine(models.Model):
                " ('order_id.state', 'in', ['draft', 'confirmed'])]")
 
     @api.multi
+    def has_consistent_route(self):
+        self.ensure_one()
+        if self.route_id and self.sourced_by:
+            selected_route = self.route_id
+            self.set_route_form_so()
+            return selected_route == self.route_id
```
Comment (gurneyalex) [inline sale_quotation_sourcing/model/sale_order.py:84]: this is a public API function, it should not be possible for it to return `None`. Could we have an explicit `return False`?

Comment (lepistone) [inline sale_quotation_sourcing/model/sale_order.py:84]: we could

**Comment by gurneyalex** [comment]
:+1:

**Comment by jgrandguillaume** [comment]
Re-tested and :+1:

**Pull request merged**